File: ZezeJava/ZezexJava/client/src/Zezex/Linkd/ModuleLinkd.java
package Zezex.Linkd;

public class ModuleLinkd extends AbstractModule {
    public void Start(Client.App app) throws Throwable {
    }

    public void Stop(Client.App app) throws Throwable {
    }

    @Override
    protected long ProcessKeepAlive(Zezex.Linkd.KeepAlive p) {
        return Zeze.Transaction.Procedure.NotImplement;
    }

    // ZEZE_FILE_CHUNK {{{ GEN MODULE @formatter:off
    public ModuleLinkd(Client.App app) {
        super(app);
    }
    // ZEZE_FILE_CHUNK }}} GEN MODULE @formatter:on
}

File: ZezeJava/ZezexJava/client/src/Zezex/Linkd/ModuleLinkd.cs

namespace Zezex.Linkd
{
    public partial class ModuleLinkd : AbstractModule
    {
        public void Start(global::Game.App app)
        {
        }

        public void Stop(global::Game.App app)
        {
        }

        protected override async System.Threading.Tasks.Task<long> ProcessKeepAlive(Zeze.Net.Protocol _p)
        {
            var p = _p as KeepAlive;
            return Zeze.Transaction.Procedure.NotImplement;
        }

    }
}
